indexing-service/src/main/java/io/druid/indexing/overlord/supervisor/SupervisorManager.java

```diff
@@ -165,7 +165,8 @@ public boolean resetSupervisor(String id, @Nullable DataSourceMetadata dataSourc
 
   public boolean checkPointDataSourceMetadata(
       String supervisorId,
-      int taskGroupId,
+      @Nullable String taskId,
+      Integer taskGroupId,
```

Comment:
should taskGroupId be nullable instead of taskId ?

Reply:
Thanks. Fixed.